src/realm/object-store/util/scheduler.hpp

```diff
@@ -103,6 +103,11 @@ class Scheduler {
     /// notifications and does not perform any thread checking.
     static std::shared_ptr<Scheduler> make_frozen(VersionID version);
 
+    /// Create a dummy scheduler which does not support any scheduler
+    /// functionality. This should be used only for Realms opened internally
+    /// and not exposed in SDKs.
+    static std::shared_ptr<Scheduler> make_dummy();
```

Comment:
Audit realms want to use the dummy scheduler too so I moved it to be with the others.